tests/ImageSharp.Tests/Formats/Jpg/JpegImagePostProcessorTests.cs

```diff
@@ -71,15 +72,70 @@ public void DoProcessorStep<TPixel>(TestImageProvider<TPixel> provider)
             }
         }
 
+        [Theory]
+        [WithFile(TestImages.Jpeg.Baseline.Calliphora, PixelTypes.Rgba32)]
+        [WithFile(TestImages.Jpeg.Baseline.Testorig420, PixelTypes.Rgba32)]
+        public void DoProcessorStepPdfJs<TPixel>(TestImageProvider<TPixel> provider)
```

Comment:
`VerifySpectralCorrectness_PdfJs/Golang` should test the first step, here we only need to test `JpegImagePostProcessor`, so parsing input with both decoders doesn't add value here. 
It's totally fine to run `JpegImagePostProcessorTests` with the default jpeg decoder only, which is `PdfJsDecoderCore` now.